test/performance/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+import com.moowork.gradle.node.yarn.YarnTask
+
+plugins {
+    id("com.cognifide.aem.common")
+    id("com.moowork.node")
+}
+
+description = "Example - Performance Tests"
+
+aem {
+    tasks {
+        register<YarnTask>("run") {
+            dependsOn("yarn")
+            group = "check"
+
+            val baseUrl = props.string("test.baseUrl") ?: main.environment.hosts.publish.url
+            val configName = "${baseUrl.substringAfter("://")}.conf"
+            val configTpl = file("sites/$configName")
+                    .takeIf { it.exists() } ?: file("sites/default.conf")
+            val configFile = file("build/sites/$configName")
+            val reportsDir = file("build/reports")
+
+            setWorkingDir(projectDir)
+            setYarnCommand("lighthouse-batch")
+            setArgs(listOf("--file", "$configFile", "--html", "--out", "$reportsDir"))
+
+            doFirst {
```

Comment:
this makes possible to run tests on any host because we have no hardcoded base URl / host name in configuration

Reply:
we are generating sites configuration file for lighthouse-batch  just before running test